pylintrc

```diff
@@ -154,6 +154,8 @@ disable=print-statement,
         deprecated-sys-function,
         exception-escape,
         comprehension-escape,
+        too-many-arguments,
+        too-many-instance-attributes,
```

Comment:
not related to this PR, but was getting pylint errors

Reply:
Is master CI failing in this case? Probably makes sense to fix the regression or revert the breaking commits rather than drop coverage on these lint rules.

Reply:
as explained in another comment, we aren't dropping the coverage on these.